src/main/kotlin/com/atlassian/performance/tools/infrastructure/database/MySql.kt

```diff
@@ -0,0 +1,65 @@
+package com.atlassian.performance.tools.infrastructure.database
+
+import com.atlassian.performance.tools.infrastructure.api.docker.DockerContainer
+import com.atlassian.performance.tools.infrastructure.api.os.Ubuntu
+import com.atlassian.performance.tools.ssh.api.SshConnection
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import java.time.Duration
+import java.time.Instant
+
+internal object MySql {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+    private val ubuntu = Ubuntu()
+
+    /**
+     * Arguments based on [jira docs](https://confluence.atlassian.com/adminjiraserver/connecting-jira-applications-to-mysql-5-7-966063305.html).
+     *
+     * We skip setting `--sql-mode` even though the docs says:
+     * "Ensure the sql_mode parameter does not specify NO_AUTO_VALUE_ON_ZERO".
+     * It's unclear what value should be set and the based on [mysql docs](https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html)
+     * the defaults are good.
+     */
+    private val jiraDocsBasedArgs = arrayOf(
+        "--default-storage-engine=INNODB",
+        "--character-set-server=utf8mb4",
+        "--innodb-default-row-format=DYNAMIC",
+        "--innodb-large-prefix=ON",
+        "--innodb-file-format=Barracuda",
+        "--innodb-log-file-size=2G"
+    )
+
+    fun installClient(ssh: SshConnection): SshSqlClient {
+        ubuntu.install(ssh, listOf("mysql-client"))
+        return SshMysqlClient()
+    }
+
+    fun container(
+        dataDir: String,
+        bonusParameters: Array<String>,
+        bonusArguments: Array<String>
```

Comment:
Unless "bonus" has better semantics, we can go for consistency with `JiraJvmArgs.extra`:
```suggestion
        extraParameters: Array<String>,
        extraArguments: Array<String>
```

Reply:
Applied this suggestion